packages/commcare/test/index.js

```diff
@@ -97,3 +97,50 @@ describe('SubmitXls', () => {
     expect(formdata.get('create_new_cases')).to.equal('on');
   });
 });
+
+describe('getCases', () => {
+  it('should fetch cases', async () => {
+    const domain = 'my-domain';
```

Comment:
Let's move these domain and app variables up to the top of the file, so all tests can share them